resources/views/_template/style.blade.php

```diff
@@ -83,15 +83,23 @@
     .table-condensed > thead > tr > td.stack {
         padding: 0;
         border-top: none;
+        background-color: #F6F6F6;
+        border-top: 1px solid #D1D1D1;
+        max-width: 0;
+        overflow-x: scroll;
```

Comment:
Understood the problem. I thought that TR itself is toggling, not only div with stack.